src/components/users/SignUpForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import Select from 'react-select';
+import { Link } from 'react-router-dom';
+
+import '../../style/App.scss';
+import './user-form.scss';
+
+const SignUpForm = () => {
+  const genders = [
+    { value: 'M', label: 'MALE' },
+    { value: 'F', label: 'FEMALE' },
+    { value: 'O', label: 'OTHER' },
+  ];
+
+  const { select_gender } = {};
+
+  const [inputs, setInputs] = useState({
+    full_name: '',
+    email: '',
+    password: '',
+    confirm_password: '',
+    gender: '',
+  });
+  const { full_name, email, password, confirm_password, gender } = inputs;
+
+  const handleChange = ({ target }) => {
+    const { name, value } = target;
+    setInputs((inputs) => ({ ...inputs, [name]: value }));
+  };
+
+  const handleChangeGender = (select_gender) => {
+    setInputs((inputs) => ({ ...inputs, ['gender']: select_gender['value'] }));
+    debugger;
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    return '';
+  };
+
+  return (
+    <form align="center" className="user-form" onSubmit={handleSubmit}>
+      <p className="user-form__text">NAME</p>
```

Comment:
import all the titles (NAME, EMAIL, PASSWORD , CONFIRM PASSWORD , GENDER) from `en.js`